helm/values.yaml

```diff
@@ -212,14 +210,18 @@ config: |
   #
   # NLB Support
```

Comment:
Should this be renamed as well?

Reply:
Well technically it's still the same in https://github.com/Telecominfraproject/wlan-cloud-ucentralgw/blob/master/ucentral.properties#L81 , so I'm not sure if we should change it here